components/collective-page/hero/OverflowCollectiveName.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
```

Comment:
I think this is a bit over-complicated. We can just use the [`truncate` function from `lodash`](https://docs-lodash.com/v4/truncate/) to achieve this, `truncate(host.name, {length: 30})`.

Reply:
Hey @SudharakaP thanks for the review! I can certainly do that, but then the tooltip would appear for both the non-truncated text as well as the truncated text (so say a name is shorter than 30 characters, it would still have a tooltip). If that's fine I'll fix this.

Reply:
The tooltip already appears right now on non-truncated text. If we do change that we probably would need to change it globally (it's coming from https://github.com/opencollective/opencollective-frontend/blob/598eaeca805b1da3922d1eade8612a7b9a0db3a4/components/Link.js#L66). Changing that probably isn't needed for now and it's not related to this issue. 

Let us wait for @Betree's thoughts on this as well.

Reply:
Gotcha, I was confused - I used `StyledTooltip` to add a non-browser tooltip (white text on black background, appears right away, rather than the delayed browser tooltip). Looks like the ask was to just use the already in place title for a browser tooltip. Can definitely fix that, it's certainly less code.

Reply:
Using a tooltip for truncated texts is not something that we do today, but I personally think that it looks nice. After digging a bit, I found multiple sources like https://www.patternfly.org/v4/ux-writing/truncation/ that promote using a tooltip for that.

So I think we can go ahead and try this solution, but we should then make this component generic (eg. `TruncatedTextWithTooltip`).